src/widgets.ts

```diff
@@ -125,6 +128,29 @@ export abstract class Widget<
     })
   }
 
+  handleOAuthRedirect(redirectURL: string): void {
```

Comment:
Nit: can we rename this to `handleOAuthComplete` to make the type?

Reply:
Also, why wound't we make this a private method?

Reply:
I see you're using this method in the tests, I think we should trigger it with a post message instead.

Reply:
>Nit: can we rename this to handleOAuthComplete to make the type?
I'm cool with the rename, but I don't understand what you mean by "make the type"

>Also, why wound't we make this a private method?
It can be private, I'll use the post message approach